### ISSUE TITLE
/saconfig info (IP)

### ISSUE THREAD

**Issue by xDestroyer217** [TotalFreedom/TotalFreedomMod#850]
A basic add-on for /saconfig info that shows admins on said IP.

Should help with users coming from random places and admin'd.

**Comment by jjkatz613** [comment]
I object, if you do /saconfig info (admin) then you will get their saconfig entry and you can see their IP from there

**Comment by xDestroyer217** [comment]
no, please stop. I'm talkin about to show all of the users in the
superadmin list in that IP.

**Comment by tylerhyperHD** [comment]
Vouch, I consistently see this happening where 2 people have the same IP, and someone needs to fix it.

**Comment by ValenciaOrange** [comment]
Could be useful to see if a admin imposed another admin
